Refit.Tests/RequestBuilder.cs

```diff
@@ -1079,41 +1085,23 @@ public void QueryStringWithArrayCanBeFormatted()
             Assert.Equal("/query?numbers=1%2C2%2C3", uri.PathAndQuery);
         }
 
-        [Fact]
-        public void QueryStringWithArrayFormattedAsMulti()
+        [Theory]
+        [InlineData("QueryWithArrayFormattedAsMulti", "/query?numbers=1&numbers=2&numbers=3")]
+        [InlineData("QueryWithArrayFormattedAsCsv", "/query?numbers=1%2C2%2C3")]
+        [InlineData("QueryWithArrayFormattedAsSsv", "/query?numbers=1%202%203")]
+        [InlineData("QueryWithArrayFormattedAsTsv", "/query?numbers=1%092%093")]
+        [InlineData("QueryWithArrayFormattedAsPipes", "/query?numbers=1%7C2%7C3")]
```

Comment:
Can you please add one more that covers the default case calling `QueryWithArray`? This can help prevent a regression in the future.

Reply:
test case would look like this 
[InlineData("QueryWithArray", "/query?numbers=System.Int32%5B%5D")]
i'm not sure if this behaviour is used by anybody so i have doubts if it should be added. So please confirm that this is what you think makes sense

Reply:
@onovotny just in case you missed previous comment. Please confirm that adding the following test case makes sense 
[InlineData("QueryWithArray", "/query?numbers=System.Int32%5B%5D")]

Reply:
@vitalyal that test does seem strange. Instead, how about one that ensures that the url parameter formatter is called with the expected value in that scenario? Basically a test with two values -- nothing, default, and `RefitParameterFormatter` that show the execution calls an `IUrlParmateterFormatter`. You can use a mock one for verifying the flow.

Reply:
@onovotny i'm not sure i understand what you suggest, but QueryStringWithArrayCanBeFormatted test may be covering that already. That is the test that was added in scope of other feature and does some regression for approach with RefitParameterFormatter (which is default value for Query attribute)

Reply:
@onovotny in case you missed previous comment, could you please describe in more details what kind of test do you expect

Reply:
Sorry, just been swamped this week.

Reply:
> @onovotny just in case you missed previous comment. Please confirm that adding the following test case makes sense
> [InlineData("QueryWithArray", "/query?numbers=System.Int32%5B%5D")]

How would i achieve the below which weabapi complient(default FromUri): 
```csharp
[InlineData("QueryWithArray", "/query?search=MyTags&tags=1&tags=2&tags=3")]
//with
class SearchQuery
{
    public string Search {get;set;}
    public int[] Tags {get;set;}
}
```